### DISCUSSION TITLE
Branding AdGuard Home

### DISCUSSION THREAD

**Discussion by KevinErasmus1999** [AdguardTeam/AdGuardHome#5158] [General]
Hi there,
I was in contact with the AdGuard Home support team and was pointed here to ask for some help.

I want to rebrand the AdGuard Home with my company's branding.

Their message reads:
"Hello!

Thank you for reaching out to AdGuard Support!

Sure, you change the AdGuard Home brand name, logo, etc. Basically, you need to follow the GPL licence—fork the source code of AdGuard Home, and change the branding. You'd also need to change the auto update procedure, so that it installs your updates, not ours. 

Let us know should you have any additional questions. Have a great day!

Best wishes,
AdGuard Support 💚"

Is there someone who can assist me in this matter please?

**Comment by samgaw58** [comment]
Did you find a solution for this?

**Comment by ainar-g** [comment]
Apologies for the very belated response.  As far as I know, as long as the terms of the [license](https://github.com/AdguardTeam/AdGuardHome/blob/master/LICENSE.txt) aren't breached, you should be able to do this.
